### ISSUE TITLE
With sumo enabled I receive the following error "Field with the given name already exists" in the system section

### ISSUE THREAD

**Issue by snap4ward** [jeff-d/quicklab#3]
While I can fully deploy quicklabs with montoring=none, if I enable sumo monitoring, a series of errors are reported.
I will report them one at a time in separate tickets, although they may have the same solution related to local.fields

Running the following:

`terraform apply -auto-approve -var="create_network=true" -var="create_bastion=true" -var="create_cluster=false" -var="monitoring=sumo"
`

Code block:
```
resource "sumologic_field" "system" {
  for_each = var.create_bastion ? toset(local.fields.resourcedetection.system) : toset([])

  field_name = each.key
  data_type  = "String"
  state      = "Enabled"
}
```

Error:

│ Error: {"id":"Y4I1F-1Q5CB-BQ2S9","errors":[{"code":"field:already_exists","message":"Field with the given name already exists"}]}
│
│   with module.sumo["gewq"].sumologic_field.system["host.name"],
│   on modules/sumo/bastion.tf line 25, in resource "sumologic_field" "system":
│   25: resource "sumologic_field" "system" {
│
╵
╷
│ Error: {"id":"OOGFF-0VUBP-NS0MZ","errors":[{"code":"field:already_exists","message":"Field with the given name already exists"}]}
│
│   with module.sumo["gewq"].sumologic_field.system["host.id"],
│   on modules/sumo/bastion.tf line 25, in resource "sumologic_field" "system":
│   25: resource "sumologic_field" "system" {

**Comment by jeff-d** [comment]
Hello! This is a known issue, called out [here](https://github.com/jeff-d/quicklab/blob/main/docs/monitoring.md#fields).

**Comment by jeff-d** [comment]
Fixed with [2e1e138](https://github.com/jeff-d/quicklab/commit/2e1e1382129ad557c9f53b9d5c4561f5c7f57a64)

**Comment by jeff-d** [comment]
Reopening due to regression introduced in [cda310b](https://github.com/jeff-d/quicklab/commit/cda310b0ed72a6788bef3104842a97fe48b9563b).
